meetings/2024-04-24.md

```diff
@@ -28,41 +35,101 @@
+* eP: Glad to hear Eric will be involved especially because we want to rely heavily on shapes. 
 
 ### Proposal for integrating DIDs into Solid
 URL: https://github.com/solid/specification/issues/629
 
-* VB: Pinged Damon to see if he can attend a call and discuss. 
+* VB: Pinged Damon to see if he can attend a call and discuss.
+* VB: This is a duplicate of a previous issue, we can discuss it when we get damon and others involved.
+* HZ: I'm netural and just wanted to clear some outstanding issues. My poisition is that Solid should accept anything that follows some characteristics. Solid should be future proof. 
+* PAC: At W3C I'm a team contact of DID group but took that role after the spec was published. On one hand I feel responsible to be consistent across the W3C groups. I think it makes sense to consider that, DID documents share a  lot of features with WebID documents. When you look at the history of WebID, there was the FOAF profile. Later Henry Story proposed to use it to prove the identity. It's first purpose was to proof that control and verify it. It makes sense to see how those things converge. People in DID community think one should put as little as possible in the DID document. This contrasts many tendencies in WebID and FOAF to be very descriptive. 
+* HZ: I would like to see something concrete. Someone could come up with a proposal. I would not expect all the DID methods to be applicable. 
+* VB: https://github.com/solid/specification/issues/217 is a proposal, we could pick up from there.
+* HZ: There were many changes in DID since there. 
+* eP: who considers temselves experts in DID? Dmitri would be a good person to invite. Jackson Morgan. We should try a STM to get the people who consider themselves eperts and discuss with them. 
+* VB: We can ask on the mailing list and take it from there.
+* SC: While there are various individuals working on both, I think it's on them to make the case. We have limited bandwidth to dedicate to features. This is the incubation space, we shouldn't wait for the working group and try to bring it directly there out of thin air. We can make a call, inviting people to make a PR, whether like they want to change WebID to DID, or propose new method, or whatever.
+* eP: Would like to emphasize designing solid in a way tht is future-proof. wWe can do our best to make it easier to introduce in the future. We want to avoid breaking changes in the future.
+* SC: If something is preventing the use of DIDs in Solid, the proponents of DID should make that case. I don't see why we need to do that work.
+* eP: We need to be careful how we evaluate people are not bringing things into the community. Some people might think it's painful/not productive and prefer to work outside. Just because they don't bring to CG doesn't mean they don't care.
```

Comment:
```suggestion
* eP: We need to be careful how we evaluate people not bringing things into the community. Some people might think it's painful or not productive, and prefer to work outside. Just because they don't bring to CG doesn't mean they don't care.
```